### ISSUE TITLE
Community Discussion: Where should we announce updates to ripme?

### ISSUE THREAD

**Issue by metaprime** [4pr0n/ripme#404]
Where should we post about updates to ripme so that people get news and visibility on new versions (specifically version 1.4, because anyone using 1.3 or below will not automatically get future updates).

**Comment by Hrxn** [comment]
Not sure...

With an account on GitHub it's simple, just put this repo on your watch list.

**Comment by metaprime** [comment]
Fair enough - and I imagine a fair amount of users are aware of the GitHub page. But I'm also guessing there's a lot of users who aren't aware or wouldn't know to look here -- or maybe even gave up on ripme during the window where it was decaying with no maintainers. I'd like to let people know it's okay to come back. Maybe there's a good place on Reddit to post where it wouldn't be considered negatively as self-promotion.

**Comment by Korons** [comment]
> Maybe there's a good place on Reddit to post where it wouldn't be considered negatively as self-promotion.

You could make your own sub for the announcements or post to [r/datahoader](https://www.reddit.com/r/DataHoarder/) [r/linux](https://www.reddit.com/r/linux) also seems to be pretty ok with open source projects posting

Github pages could also work

**Comment by coder966** [comment]
You may also consider a Google+ community. A lot of Android Apps have users' community as well as some Linux distributions.

**Comment by metaprime** [comment]
@Korons Creating a GitHub Pages site for this repo is a good idea, regardless. (However in order for people to see updates there they'd have to know about it and go looking for it.)

**Comment by Matt07211** [comment]
How about create a GitHub, and have the application pull down the webpage data every couple of hours,  then alert.
This probably could be implemented such as;
- Create a GitHub pages, then an update section 
- Create an update section RSS feed
- The application will then regularly check the RSS and specific intervals 
- User gets notified if a new versions is available (can also provide a changelog at the area where users see the new app update available area).

This allows you to just update the GitHub page as needed.

This is just an idea.

**Comment by metaprime** [comment]
@Matt07211 I think we've got updates and changelog covered already. The news feed is not a bad idea but I feel like that might be perceived as spamming within the app. More than what people are looking for. I guess I was hoping to stick a link under the help menu or something so people can easily see where to go to find out more about the app.

**Comment by Matt07211** [comment]
To each their own. As long as you have a link to GitHub, people can update as needed. :)

**Comment by metaprime** [comment]
I was thinking the app needs a Help menu. In that menu, links to our GitHub Wiki, for example to a Getting Started page, link to create a new issue with a template to fill in or something, etc. Definitely would be possible to link to a news feed about the app as well, and to the changelog (which is basically just `ripme.json` although that's more of a summary format of the changes and not as exhaustive as a changelog maybe should be.

**Comment by metaprime** [comment]
https://github.com/4pr0n/ripme/issues/419

**Comment by Matt07211** [comment]
As I said in another comment. Yep that sounds good.
